kuma/javascript/src/components/section-link.jsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
```

Comment:
Let's not use the folder "components". This file isn't even a component. It's a little helper function.

Reply:
Agreed. I missed this one when I changed some of this earlier today